src/index.ts

```diff
@@ -928,11 +929,30 @@ class NixInstallerAction extends DetSysAction {
     try {
       await access(receiptPath);
       // There is a /nix/receipt.json
+      actionsCore.info("\u001b[32m Found /nix/receipt.json \u001b[33m");
```

Comment:
A few edits here (and following):

```suggestion
      actionsCore.info("\u001b[32m Nix is already installed: found /nix/receipt.json \u001b[33m");
```